lib/services/bundleService.ts

```
@@ -0,0 +1,126 @@
+ */
+export function processBundleData(bundleData: any): any[] {
+  if (!bundleData || !bundleData.aggregations || !Array.isArray(bundleData.aggregations)) {
+    console.warn('Invalid bundle data structure', bundleData);
+    return [];
+  }
+
+  // Extract unique bundles
+  const bundleMap = new Map();
+  
+  bundleData.aggregations.forEach((item: any) => {
+    if (item.selectors && item.selectors["Bundle Description"]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)